alert.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+			:host {
+				--very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 0.25rem;
+                --border-radius: 0.15rem;
+                --padding: 0.3rem;
+			}
+								
+            .vui-alerts-alert {			
```

Comment:
You can likely get away without the surrounding `<div>` and this class. So this would just become:

    :host {
        /* all those styles */
    }